docs/maintenance.md

```diff
@@ -0,0 +1,51 @@
+# Maintenance
+
+## How to update supported Kubernetes
+
+Nyamber supports the three latest Kubernetes versions.
+If a new Kubernetes version is released, please update the followings:
+
+### 1. Update supported kubernetes and dependencies versions
+
+- Kubernetes versions: You can check the available versions at <https://hub.docker.com/r/kindest/node/tags>.
```

Comment:
I think this link `https://hub.docker.com/r/kindest/node/tags` is used to determine which version to specify for `k8s-version` in `.github/workflows/ci.yaml`. So I think it should be changed as follows:
```suggestion
- Kubernetes versions:
  - `k8s-version` in [.github/workflows/ci.yaml](/.github/workflows/ci.yaml): You can check the available versions at <https://hub.docker.com/r/kindest/node/tags>.
```

Reply:
Exactly, thanks!